<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>种植体多孔优化</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/Implant.png</normaloff>../icons/Implant.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #f0f2f5
}

QWidget {
	border: 1px solid #C2C7CB;
}

Line {
	color: #AA0000;
	background-color: solid #e0e0e0;
}

QToolButton {
	background-color: #A6A6A6;
	border: 1px solid;
	border-radius: 3px;
}

QToolButton:hover {
    background-color: #E6E6E6;
}

QToolButton:checked {
	background-color:  qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #A6A6A6, stop:1 #F8B177);
	border:2px solid #C55D08;
}

QPushButton {
	background-color: #A6A6A6;
	border: 1px solid;
	border-radius: 3px;
}

QPushButton:hover {
    background-color: #E6E6E6;
}

@keyframes gradient {
    0% { background-position: 0% 50%; }
    50% { background-position: 100% 50%; }
    100% { background-position: 0% 50%; }
}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="3" column="2" colspan="2">
     <widget class="QLineEdit" name="workPathLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="hmButton">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>35</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>初始.hm文件</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icons/floder.png</normaloff>../icons/floder.png</iconset>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QPushButton" name="beamButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>梁径优化设置</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icons/setting-50.png</normaloff>../icons/setting-50.png</iconset>
      </property>
     </widget>
    </item>
    <item row="7" column="2" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>35</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="addButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>35</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/add.png</normaloff>../icons/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="deleteButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>35</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/remove.png</normaloff>../icons/remove.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="structureButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>结构优化设置</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icons/setting-50.png</normaloff>../icons/setting-50.png</iconset>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="4">
     <widget class="CTable" name="conditionTableWidget">
      <column>
       <property name="text">
        <string>工况名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>工况设置</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="12" column="0" colspan="4">
     <widget class="Line" name="line_3">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="workPathButton">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>运行目录</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icons/floder.png</normaloff>../icons/floder.png</iconset>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="4">
     <widget class="Line" name="line_2">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="13" column="1" colspan="3">
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="runBeamButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>35</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>运行：梁径优化</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/run.png</normaloff>../icons/run.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QToolButton" name="preferenceButton">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>35</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>预设</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icons/setting-50.png</normaloff>../icons/setting-50.png</iconset>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="Line" name="line_1">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="conditionButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>载荷设置</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icons/setting-50.png</normaloff>../icons/setting-50.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLineEdit" name="hmLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="10" column="2" colspan="2">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="runStructureButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>35</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>运行：结构优化</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/run.png</normaloff>../icons/run.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="11" column="0" colspan="4">
     <widget class="QTableWidget" name="structureTable"/>
    </item>
    <item row="14" column="0" colspan="4">
     <widget class="QTableWidget" name="beamTable"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTable</class>
   <extends>QTableWidget</extends>
   <header>./GUI/widgets/cTable</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
